doc/users/whatsnew/multiple_legend_keys.rst

```diff
@@ -6,11 +6,9 @@ A legend entry can now contain more than one legend key. The extended
 legend area in the specified number of sections; ``pad`` changes the padding
 between the legend keys.
 
-.. figure:: ../../gallery/pylab_examples/images/sphx_glr_legend_demo6_001.png
-   :target: ../../gallery/pylab_examples/legend_demo6.html
+.. figure:: /gallery/text_labels_and_annotations/images/sphx_glr_legend_demo_004.png
```

Comment:
PS, @choldgraf for these, we don't need the `../..` because Sphinx accepts an absolute path. Unfortunately, it doesn't work on the target below though.